### PR TITLE
ui: Fix route to ISO From VM's Info Card / Detail View

### PR DESCRIPTION
**Pull request by Pearl1594**
### Description

This PR fixes the route to Iso tab from the respective VM's detail View (Info card) if the VM is deployed from an ISO. 

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [X] Minor
- [ ] Trivial

### How Has This Been Tested?
Deploy VM from an ISO and traverse to the ISO tab from the detail view
Before Fix:
image

Post Fix:
Successfully able to route to iso tab

### CODE REVIEW

**Comment by Pearl1594** [comment]
@blueorangutan ui

**Review by utchoang — APPROVED** [review]
LGTM

**Review by shwstppr — APPROVED** [review]

File: ui/src/components/view/InfoCard.vue [hunk 1]
```diff
@@ -398,7 +398,12 @@
           <div class="resource-detail-item__label">{{ $t('label.templatename') }}</div>
```
Comment (shwstppr) [inline ui/src/components/view/InfoCard.vue:398]: @Pearl1594 should we also have label read ISO with something like
```suggestion
          <div class="resource-detail-item__label">{{ resource.isoid ? $t('label.iso') : $t('label.templatename') }}</div>
```

Comment (Pearl1594) [reply ui/src/components/view/InfoCard.vue]: Thanks @shwstppr. Makes sense.. I've updated the PR.

**Comment by Pearl1594** [comment]
@blueorangutan ui

**Review by shwstppr — APPROVED** [review]

**Pull request merged**